include/circt/Dialect/FIRRTL/InnerSymbolTable.h

```diff
@@ -182,6 +182,30 @@ class InnerSymbolTable {
   /// A successful walk with no failures returns success.
   static LogicalResult walkSymbols(Operation *op, InnerSymCallbackFn callback);
 
+  /// Get the sym name from the InnerSymPropertiesAttr.
+  static StringAttr getInnerSymName(const InnerSymPropertiesAttr p) {
+    return p.getName();
+  }
+
+  /// Get the iterator for all the symbol names for an InnerSymAttr.
+  static auto getSymNamesBegin(InnerSymAttr isa) {
```

Comment:
This doesn't seem like the place for these methods, but I'm not sure where they should live.  Member functions on the attribute seems ideal, if tablegen doesn't make that too painful.